.github/workflows/deploy.yml

```
@@ -0,0 +1,57 @@
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          workingDirectory: 'apps/web-main'
+          command: pages deploy .next --project-name=web-main-prod
+
+      - name: Deploy Prosper Web (Pages)
+        uses: cloudflare/wrangler-action@v3
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          workingDirectory: 'apps/web-prosper'
+          command: pages deploy .next --project-name=prosper-web
+d
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)